packages/framepost/.eslintrc.js

```diff
@@ -0,0 +1,374 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+module.exports = {
+    root: true,
+    globals: {
+        it: true,
+        expect: true,
+        test: true
+    },
```

Comment:
I think this is the only bit that's actually different from the top-level `.eslintrc.js`. We _should_ be able to remove everything except this section from this file and have it all stil workl.
```Console
$ diff --unified .eslintrc.js packages/framepost/.eslintrc.js 
--- .eslintrc.js	2021-12-21 12:19:19.000000000 -0800
+++ packages/framepost/.eslintrc.js	2021-12-21 12:19:19.000000000 -0800
@@ -1,6 +1,11 @@
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 module.exports = {
     root: true,
+    globals: {
+        it: true,
+        expect: true,
+        test: true
+    },
     rules: {
         'block-scoped-var': 'error',
         curly: ['error', 'all'],
@@ -361,9 +366,9 @@
         }
     },
     env: {
-        jest: true,
         es6: true,
         node: true,
-        browser: true
+        browser: true,
+        jest: true
     }
 };
```